### PR TITLE
(maint) retrofitting changelog for the last couple of releases

### PR DESCRIPTION
**Pull request by DavidS**
This moves the existing CHANGELOG to HISTORY.md since github_changelog_generator
is not being stable with old CHANGELOG entries.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,71 @@
...
+- \(packaging\) Update reported version to 1.8.2 \[no-promote\] [\#167](https://github.com/puppetlabs/puppet-resource_api/pull/167) ([mihaibuzgau](https://github.com/mihaibuzgau))
+
+## [v1.6.4](https://github.com/puppetlabs/puppet-resource_api/tree/v1.6.4) (2019-03-25)
+[Full Changelog](https://github.com/puppetlabs/puppet-resource_api/compare/v1.8.1...v1.6.4)
+
+**Merged pull requests:**
+
+- Add `implementations` to reserved bolt keywords [\#165](https://github.com/puppetlabs/puppet-resource_api/pull/165) ([DavidS](https://github.com/DavidS))
+- \(MAINT\) Bump version [\#164](https://github.com/puppetlabs/puppet-resource_api/pull/164) ([sebastian-miclea](https://github.com/sebastian-miclea))
+- Release prep for v1.8.1 [\#163](https://github.com/puppetlabs/puppet-resource_api/pull/163) ([DavidS](https://github.com/DavidS))
```
Comment (da-ar) [inline CHANGELOG.md]: this looks redundant

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,71 @@
...
+## [1.8.3](https://github.com/puppetlabs/puppet-resource_api/tree/1.8.3) (2019-04-12)
+[Full Changelog](https://github.com/puppetlabs/puppet-resource_api/compare/1.8.2...1.8.3)
+
+**Fixed bugs:**
+
+- \(FM-7867\) Always throw when transport schema validation fails [\#169](https://github.com/puppetlabs/puppet-resource_api/pull/169) ([da-ar](https://github.com/da-ar))
+
+**Merged pull requests:**
+
+- \(PA-2496\) Bump version and remove v from version number [\#170](https://github.com/puppetlabs/puppet-resource_api/pull/170) ([mihaibuzgau](https://github.com/mihaibuzgau))
```
Comment (da-ar) [inline CHANGELOG.md]: should this be recorded here?

Comment (DavidS) [reply CHANGELOG.md]: I can tag the other's `maintenance`, but this should really stay, as it is a user-visible change

**Comment by DavidS** [comment]
Labled a bunch of merged PRs and re-ran the generator

**Comment by DavidS** [comment]
Cleared out my local github_changelog_generator cache for better results.

**Comment by DavidS** [comment]
Rebased on top of current master.

**Pull request merged**